volume-providers/vpc/provider/util.go

```diff
@@ -41,9 +42,19 @@ var skipErrorCodes = map[string]bool{
 	"invalid_route":                    false,
 }
 
+func getEnv(key string) string {
+	return os.Getenv(strings.ToUpper(key))
+}
+
 // retry ...
 func retry(logger *zap.Logger, retryfunc func() error) error {
 	var err error
+	if vpcRetryInterval := getEnv("VPC_RETRY_INTERVAL"); vpcRetryInterval != "" {
```

Comment:
Lest use this method to get all retry parameters --->  https://github.com/IBM/ibmcloud-storage-volume-lib/blob/master/config/config.go#L179